pkgs/by-name/su/surge-XT/package.nix

```diff
@@ -27,6 +27,23 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-4b0H3ZioiXFc4KCeQReobwQZJBl6Ep2/8JlRIwvq/hQ=";
   };
 
+  patches = [
+    # NOTE: merged in upstream, remove on package update
+    #     (https://github.com/surge-synthesizer/surge/pull/8202)
+    ./clap-option.diff
+  ];
```

Comment:
I'm sorry @mrtnvgr for being so harsh, but the `clap-option.diff` patch does not belong to this commit. Also, the commit that adds it to the expression, should be the commit that adds the `diff` file itself.

Reply:
No, that's okay. Learning is fun 😊